espnet2/train/preprocessor.py

```diff
@@ -505,19 +505,24 @@ def __init__(
         self,
         train: bool,
         source_scp: str = None,
-        num_spk: int = 2,
+        num_utts: int = 2,
```

Comment:
Maybe `num_refs` is better, if we want to add a new preprocessor for meeting-style data in the future.

Reply:
I changed to ref_num, same as the one used in enh.sh. Trying to reduce the similar confusing names across all files.